src/plugins/empty-org-recycle-bin/enable.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
When you start developing the tests,

- `enable.json` could be renamed to `empty-org-recycle-bin.json`
- `disable.json` could probably be removed